### ISSUE TITLE
Label size specified programmatically are not in sp

### ISSUE THREAD

**Issue by ispbox** [skydoves/ProgressView#2]
If I specify label size from code, it looks like it is measured in pixels. Here is the code from library:
```
this.labelView.setTextSize(TypedValue.COMPLEX_UNIT_PX, labelSize)
```
I can propose to change it to 
```
this.labelView.setTextSize(labelSize)
```
because this way you will specify size in sp, not in pixels. This is how all controls in android work.  Your default value of 12f is very small because of px units instead of sp.

**Comment by ispbox** [comment]
Because of this I have to specify label size in such a strange manner:
labelSize = 14f * getResources().getDisplayMetrics().scaledDensity

**Comment by skydoves** [comment]
@ispbox 
Hi, thank you for your issue!
It will be included on next version soon.
Thank you :)

**Comment by ispbox** [comment]
Thanks a lot :)

**Comment by skydoves** [comment]
@ispbox 
Released version [1.0.2](https://github.com/skydoves/ProgressView/releases/tag/1.0.2).
Thank you for your issue.

**Comment by ispbox** [comment]
@skydoves Thanks a lot, works good :)
